util.js

```diff
@@ -609,58 +609,80 @@ function hasOwnProperty(obj, prop) {
 
 var kCustomPromisifiedSymbol = typeof Symbol !== 'undefined' ? Symbol('util.promisify.custom') : undefined;
 
-exports.promisify = function promisify(original) {
-  if (typeof original !== 'function')
+exports.promisify = getPromisify();
+
+exports.promisifyFn = getPromisify('promisifyFn');
```

Comment:
This package is for node's `util` module, in browsers. `require('util').promisifyFn` doesn't exist, and so this here must not exist.